### ISSUE TITLE
[ngtools] exported services/pipes are removed from AOT build

### ISSUE THREAD

**Issue by QuentinFchx** [angular/angular-cli#8845]
### Versions
```
@angular/core: ^5.1.0
@ngtools/webpack: ^1.9.0
```

### Repro steps

[Code sample](https://github.com/QuentinFchx/aot-export) `npm run build:aot`
* Step 1
Write a Pipe/Service and declare it in a module
* Step 2
Export the Pipe/Service from the module
* Step 3
Use the exported service anywhere else (ie: angularJS app through `downgradeInjectable(HelloService)`)

### Observed behavior
It's impossible to export a service and use it in an angularJS application through `downgradeInjectable`
```js
// webpack output
Object.defineProperty(exports, "__esModule", { value: true });
var AppModule = /** @class */ (function () {
    function AppModule() {
    }
    return AppModule;
}());
exports.AppModule = AppModule;
```

### Desired behavior
The service should be exported.
```js
// webpack output
Object.defineProperty(exports, "__esModule", { value: true });
var tslib_1 = __webpack_require__(0);
var core_1 = __webpack_require__(2);
var platform_browser_1 = __webpack_require__(18);
var AppPage_1 = __webpack_require__(50);
var HelloPipe_1 = __webpack_require__(51);
exports.HelloPipe = HelloPipe_1.HelloPipe;
var HelloService_1 = __webpack_require__(19);
exports.HelloService = HelloService_1.HelloService;
var AppModule = /** @class */ (function () {
    function AppModule() {
    }
    AppModule = tslib_1.__decorate([
        core_1.NgModule({
            imports: [platform_browser_1.BrowserModule],
            providers: [HelloService_1.HelloService],
            declarations: [AppPage_1.AppPage, HelloPipe_1.HelloPipe],
            exports: [HelloPipe_1.HelloPipe],
            bootstrap: [AppPage_1.AppPage]
        })
    ], AppModule);
    return AppModule;
}());
exports.AppModule = AppModule;
```

### Mention any other details that might be useful (optional)
About pipes: I could not find any "best practice" on how to use pipes in code ([community answer1](https://stackoverflow.com/questions/35159079/is-it-possible-to-use-pipe-in-the-code-in-angular-2), [community answer2](https://stackoverflow.com/questions/35144821/angular-2-4-use-pipes-in-services-and-components)). This is not related to `angular-cli` but someone might have the answer here? To work around this issue (the pipe removal), for now I just use an intermediate function that I call in the pipe and that I import in my code. (But I cannot apply this hack to Injectables)
```ts
import { Pipe, PipeTransform } from '@angular/core';

export function hello(){ return 'Hello' }

@Pipe({ name: 'hello' })
export class HelloPipe implements PipeTransform {
    transform(val: string): string { return hello(); }
}
```

**Comment by QuentinFchx** [comment]
Hi @hansl sorry to bother you.

Would it be possible to tell me if it's a real issue within the `@ngtools/webpack` package? If it is, I _might_ have the time to help fixing it.

Otherwise could you point me in the right direction to properly handle the use-cases I mentionned above (re-exported pipes/services)?

Thank you in advance 😊

**Comment by QuentinFchx** [comment]
It seems that a recent update in either @ngtools/webpack or @angular fixed the issue.
Thank you and sorry for the inconvenience 😄

**Comment by QuentinFchx** [comment]
Fixed by #9233
